packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/AnnotationEditorUtils.ts

```diff
@@ -115,65 +121,51 @@ export function getFriendlyPropertyName(error: RJSFValidationError): string {
   }
 }
 
-export function getTransformErrors(concreteType?: ENTITY_CONCRETE_TYPE) {
-  return (errors: RJSFValidationError[]): RJSFValidationError[] => {
-    // Transform the errors in the following ways:
-    // - Simplify the set of errors when failing to select an enumeration defined with an anyOf (SWC-5724)
-    // - Show a custom error message when using a property that collides with an internal entity property (SWC-5678)
-
-    // Fixing anyOf errors
-    // Group the errors by the property that the error applies to
-    const grouped = groupBy(errors, error => error.property)
-    Object.keys(grouped).map(property => {
-      const errorGroup = grouped[property]
-
-      // First, see if it is an anyOf error
-      const hasAnyOfError = errorGroup.some(
-        e => e.message === 'should match some schema in anyOf',
-      )
-
-      // We determine if it's an anyOf *enum* error if all error messages in the property match one of these three messages:
-      const isEnumError =
-        hasAnyOfError &&
-        errorGroup.every(error => {
-          if (error.message === 'should be string') {
-            return true
-          } else if (error.message === 'should be equal to constant') {
-            return true
-          } else if (error.message === 'should match some schema in anyOf') {
-            return true
-          } else {
-            return false
-          }
-        })
-
-      // If it's an anyOf enum error, we just modify the first message and drop the rest
-      if (isEnumError && errorGroup.length > 0) {
-        errorGroup[0].message = 'should be equal to one of the allowed values'
-        grouped[property] = [errorGroup[0]]
-      }
-    })
-
-    // Ungroup the errors after potentially modifying them
-    errors = flatMap(grouped)
-
-    // Custom error message if the custom annotation key collides with an internal entity property
-    if (concreteType) {
-      errors = errors.map(error => {
-        const propertyName = getFriendlyPropertyName(error)
-        if (
-          propertyName &&
-          entityJsonKeys[concreteType].includes(propertyName)
-        ) {
-          error.message = `"${propertyName}" is a reserved internal key and cannot be used.`
+export function transformErrors(
+  errors: RJSFValidationError[],
+): RJSFValidationError[] {
+  // Transform the errors in the following ways:
+  // - Simplify the set of errors when failing to select an enumeration defined with an anyOf (SWC-5724)
+  // - Show a custom error message when using a property that collides with an internal entity property (SWC-5678)
+
+  // Fixing anyOf errors
+  // Group the errors by the property that the error applies to
+  const grouped = groupBy(errors, error => error.property)
+  Object.keys(grouped).map(property => {
+    const errorGroup = grouped[property]
+
+    // First, see if it is an anyOf error
+    const hasAnyOfError = errorGroup.some(
+      e => e.message === 'should match some schema in anyOf',
```

Comment:
Are these RJSF validation error messages exported?  It would be preferable to reference a constant from the library (in case it changes in the future).  
This is obviously not a new issue in this changeset, and does not block merge.